sentenai/historiQL.py

```diff
@@ -765,14 +868,20 @@ def __eq__(self, val):
         Arguments:
             val -- The value to compare the stream variable to.
         """
-        if type(val) == list:
-            return Cond(self, 'in', val)
+        if isinstance(val, CondChain):
+            val.lcond = Cond(self, 'in' if type(val.lval) is list else '==', val.lval)
+            val.lpath = self
+            return val.reify()
         else:
-            return Cond(self, '==', val)
+            return Cond(self, 'in' if type(val) is list else '==', val)
+
+    def __contains__(self, path):
+        joiner = "$$$$%$$$$"
+        return joiner.join(self._attrlist) in joiner.join(self._attrlist)
```

Comment:
this looks like a typo (which is copy-pasted down below. `path` is unused and `joiner.join(self._attrlist)` is called twice